examples/wind-polar-selections.py

```diff
@@ -0,0 +1,96 @@
+from dash import Dash, html, dcc, callback, Input, Output
+import dash_bootstrap_components as dbc
+import plotly.express as px
+
+app = Dash(__name__, external_stylesheets=[dbc.themes.CYBORG])
+
+df = px.data.wind()
+df[["min_strength", "max_strength"]] = (
+    df.strength.str.replace(r"6\+", "6-7").str.split("-", expand=True).astype(int)
```

Comment:
When I run this locally, I get a future warning
> The default value of regex will change from True to False in a future version.


Adding ` regex=True` seems to fix it.

Reply:
made `regex=False` as it was actually not needed here.